digitalstrategie/assets/scss/_variables.scss

```diff
@@ -11,6 +11,7 @@ $indigo:  #6610f2 !default;
 $purple:  #aca4d1 !default; // from ds design
 $pink:    #f5b4cb !default; // from ds design
 $red:     #dc3545 !default;
+$red20:   #dc354520 !default; // added
```

Comment:
I think, you can change the red that's already there and then just make a note that it's from the design now!